manifests/p/PopeenCom/ClassicVolumeMixer/2.2/PopeenCom.ClassicVolumeMixer.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: PopeenCom.ClassicVolumeMixer
+PackageVersion: 2.2
+PackageLocale: en-US
+Publisher: PopeenCom
+PackageName: Classic Volume Mixer
+License: MIT License
+ShortDescription: Classic Volume Mixer Setup
```

Comment:
Please provide a better short description of what Classic Volume Mixer does

Reply:
Done